### PR TITLE
Adds VVD-40 spray.

### PR DESCRIPTION
**Pull request by Omni44**
## About The Pull Request

It's just a spray can with 50u of rejuvenating agent with it, bit less common than combitool. That's it, really.

## Why It's Good For The Game

Figured that rejuvenating agent is kinda hard to get without chem's help (at least as far as I tried in practise) or setting your own lab up.

## Changelog
:cl:
add: Added VVD-40 and sprite for it.
/:cl:

### CODE REVIEW

**Comment by Omni44** [comment]
Travis why

File: code/modules/reagents/reagent_containers/spray.dm [hunk 1]
```diff
@@ -193,4 +193,14 @@
...
+	name = "WD-40"
+	desc = "Relic of ancient times, rumoured to fix anything duct tape can't."
+	icon = 'icons/obj/tools.dmi'
+	icon_state = "rejuvenating_agent_spray"
+	item_state = "rejuvenating_agent_spray"
+	volume = 100
+	preloaded_reagents = list("rejuvenating_agent" = 50)
+	rarity_value = 40
+	spawn_tags = SPAWN_TAG_TOOL
 	..()
```
Comment (MLGTASTICa) [inline code/modules/reagents/reagent_containers/spray.dm:206]: random ..() , youd know if you actually followed guidelines and tried to not do web-edits.

Comment (Omni44) [reply code/modules/reagents/reagent_containers/spray.dm:206]: But it was already there, I didn't touch it,

Comment (MLGTASTICa) [reply code/modules/reagents/reagent_containers/spray.dm:206]: You moved it from the proc above that handles spray damage ,your item is practically inside that PROC and screaming out that it doesn't work

File: code/modules/reagents/reagent_containers/spray.dm [hunk 2]
```diff
@@ -192,5 +192,16 @@
 
 	if(istype(A, /obj/effect/blob)) // blob damage in blob code
 		return
-
 	..()
+
+
+/obj/item/weapon/reagent_containers/spray/WD40
```
Comment (Firefox13) [inline code/modules/reagents/reagent_containers/spray.dm:198]:
```suggestion
/obj/item/weapon/reagent_containers/spray/wd40
```
Lowercase in paths is enforced for the most part

**Comment by Omni44** [comment]
I got trolled, big time. My hat is off.
Anyways, should be working now.

**Comment by TheShown911** [comment]
I don't like this, restoration of vagabond loot should be left for non vagabonds.

**Comment by drexample** [comment]
I like it, BUT
1. Name should be a reference, not a direct name drop
2. Make it have an extremely small amount of the stuff. Barely enough to use it once, maybe even less. This thing has been there for a while and is sought-after.

**Comment by Wouju** [comment]
> 1. Name should be a reference, not a direct name drop

Agreed
> 2. Make it have an extremely small amount of the stuff. Barely enough to use it once, maybe even less. This thing has been there for a while and is sought-after.

that's one of the dumbest things i've read, whats the point of it existing if it barely has enough to have any fucking use?

**Comment by drexample** [comment]
> > 1. Name should be a reference, not a direct name drop
> 
> Agreed
> 
> > 1. Make it have an extremely small amount of the stuff. Barely enough to use it once, maybe even less. This thing has been there for a while and is sought-after.
> 
> that's one of the dumbest things i've read, whats the point of it existing if it barely has enough to have any fucking use?

s c a r c i t y

**Comment by Sparrowbird611** [comment]
Also would like to point out that this shit ain't sought after in the slightest. Most people don't even know it exists.

**Comment by Sparrowbird611** [comment]
Either way, the amount is fine as is right now.

**Comment by Sparrowbird611** [comment]
> > > > Also would like to point out that this shit ain't sought after in the slightest. Most people don't even know it exists, and they are perfectly willing to use the normal guns.
> > > 
> > > 
> > > So no point in adding it?
> > 
> > 
> > No point in opposing it?
> 
> Adding something for the sake of adding something is an awful practice and people that advocate it are cringe.

Gets done all the time, sorry bout that. If you want to nitpick a little bottle of spray that slightly buffs guns, be my guest.

**Comment by Sparrowbird611** [comment]
Also please lets stop quoting, we're about to recreate the EMP pr.

**Comment by drexample** [comment]
> Also please lets stop quoting, we're about to recreate the EMP pr.

no

**Comment by Sparrowbird611** [comment]
fuck

**Comment by Omni44** [comment]
> I don't like this, restoration of vagabond loot should be left for non vagabonds.

I ran a test earlier today, went around the ship with spare for over 10 minutes to see when I find it. Before I found one can, I found 3 middle-high tier weapons, few gun disks of various kinds including two mandela ones and I am not even counting all other stuff I plainly ignored, trashpiles I plainly ignored and I'd say I went through 20% of maint tops. It's game of statistcs and luck of course, but I don't think it's *that* meaningful vagahobo buff.  To top that off, most chemists I happened to ask for it, didn't even know what I am talking about. The only times I've been able to obtain it in game, was by breaking into chem myself, even setting up small chemistry in maint myself wouldn't get me all necessary ingridients with parts I managed to obtain.  
I could argue against some thigns I don't like about lathe nerf in general, despite liking it as principle, but that's not the place.

>Name should be a reference, not a direct name drop
VVD-40?
>Make it have an extremely small amount of the stuff. Barely enough to use it once, maybe even less. This thing has been there for a while and is sought-after.
I'm not honestly sure how it works, wouldn't it fix a gun even if you somehow managed to spray one unit of it?
I can reduce amount prefilled/make it even rarer, sure, but single use sounds tad unreasonable.

**Comment by Wouju** [comment]
> s c a r c i t y

bruh moment

also avoid quoting too much since its gonna be a pain scrolling/reading

**Comment by Sparrowbird611** [comment]
Name it mech healer serum. I don't care that mech healer serum is supposed to ressurect people, dew it.

**Comment by Firefox13** [comment]
This will essentially triviliaze the trashed loot because once found one of these you're golden. It also cuts balls to way to make more for Medical who are already almost literally hobos.

**Comment by drexample** [comment]
People on the ship, especially fucking dumpster divers, should have less access to resources.

**Comment by Alliostra** [comment]
Firstly there is absolutely no reason for this to be made, secondly it will screw up loot and make all other items more common, thirdly, one and possibly more item quality changes are direct violation to contributing guidelines, you should try reading them.
https://wiki.cev-eris.com/Content_GuidelinesEn

**Review by Firefox13 — APPROVED** [review]
The spray will be added as off-ship spawn once we're ~~feel like it~~ start on it. Atleast it fixes mistypos...

**Pull request merged**